encoding/thrift/register.go

```diff
@@ -103,10 +107,13 @@ func BuildProcedures(s Service, opts ...RegisterOption) []transport.Procedure {
 			Protocol:     proto,
 			Enveloping:   rc.Enveloping,
 		})
+		signature, _ := s.Signatures[methodName]
```

Comment:
No need for the blank identifier; this is the same as `signature := s.Signatures[methodName]`, so you might as well do it inline in the procedure literal below.